ocis-pkg/oidc/options.go

```diff
@@ -0,0 +1,121 @@
+package oidc
+
+import (
+	"net/http"
+
+	"github.com/owncloud/ocis/v2/ocis-pkg/log"
+	"github.com/owncloud/ocis/v2/services/proxy/pkg/config"
+
+	gOidc "github.com/coreos/go-oidc/v3/oidc"
+)
+
+// Option defines a single option function.
+type Option func(o *Options)
+
+// Options defines the available options for this package.
+type Options struct {
+	// HTTPClient to use for requests
+	HTTPClient *http.Client
+	// Logger to use for logging, must be set
+	Logger log.Logger
+	// The OpenID Connect Issuer URL
+	OidcIssuer string
```

Comment:
```suggestion
	OIDCIssuer string
```
Abbreviations should be all lower. See `JWKSOptions`